doc/codingGuidelines.md

```diff
@@ -227,8 +227,8 @@ Headers should be included in the following order, with each section separated b
 
 #include <string>
 
-#if defined(WANT_UFE_BUILD)
-  #include <ufe/ufe.h>
+#ifdef UFE_V2_FEATURES_AVAILABLE
```

Comment:
Do we still need the UFE_V2_FEATURES_AVAILABLE flag if we can only build with UFE V2 available?

Reply:
Ah right, this flag will always be available in ufe.h, but I should have used the V3 one here.